code/web/src/modules/user/Profile.js

```diff
@@ -18,82 +18,120 @@ import { logout } from './api/actions'
-        <p style={{ color: grey2, marginBottom: '2em' }}>{props.user.details.email}</p>
+      <Grid>
+        <GridCell style={{ padding: "2em", textAlign: "center" }}>
+          <img
+            src={details.image || `${APP_URL}//images/stock/men/4.jpg`}
+            style={{
+              marginBottom: "2em",
+              height: "24em",
+              boxShadow: level1,
+              borderRadius: "10px",
+            }}
+          />
+          <H4>{details.name}</H4>
+        </GridCell>
 
-        <H6 style={{ marginBottom: '0.5em' }}>Shipping Address:</H6>
-        <p style={{ color: grey2, marginBottom: '2em' }}>
-          123 Vanilla St.<br/>Chocolate, CO<br/>80911
-        </p>
+        <GridCell style={{ padding: "2em", textAlign: "center" }}>
+          <H4 font="secondary" style={{ marginBottom: "1.5em" }}>
+            Account Details
+          </H4>
 
-        <H6 style={{ marginBottom: '0.5em' }}>Bio:</H6>
-        <p style={{ color: grey2, width: '60%', margin: '0 auto 2em' }}>
-          "Clothes are just accessories for my rollerblades, and I'm all about accessories."
-        </p>
+          <H6 style={{ marginBottom: "0.5em" }}>Account email:</H6>
+          <p style={{ color: grey2, marginBottom: "2em" }}>
+            {details.email}
+          </p>
 
-        <Link to={userRoutes.editProfile.path}>
-          <Button theme="primary">Edit Profile Info</Button>
-        </Link>
-      </GridCell>
+          <H6 style={{ marginBottom: "0.5em" }}>Shipping Address:</H6>
+          <p style={{ color: grey2, marginBottom: "2em" }}>
+            {details.street && 
+              <>
+                {details.street}
+                <br />
+                {details.city}, {details.state}
+                <br />
+                {details.zip}
+              </>
+            }
+            {!details.street && 'You have not provided an address'}
+          </p>
 
-      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
-        <div 
-          style={{
-            backgroundColor: grey,
-            borderRadius: '10px',
-            padding: '20px',
-            width: '75%',
-            margin: '0 auto',
-            boxShadow: `inset -1px 1px 10px ${grey2}`
-          }}
-        >
-          <H5 style={{ color: 'black', marginBottom: '0.5em' }}>Your next crate will ship on:</H5>
-          <H6 style={{ color: secondary, marginBottom: '1em' }}>November 13, 2020</H6>
-          <Button theme="primary">Edit Ship Date</Button>
-        </div>
-
-        <img src={`${APP_URL}/images/crate.png`} style={{ height: '12em', display: 'block', margin: '2.4em auto 1em' }}/>
-        <Button theme="primary">View Order History</Button>
-      </GridCell>
-    </Grid>
-  </div>
-)
+          <H6 style={{ marginBottom: "0.5em" }}>Bio:</H6>
+          <p style={{ color: grey2, width: "60%", margin: "0 auto 2em" }}>
+            {details.bio ? `"${details.bio}"` : "You have not written a bio"}
```

Comment:
Very cool that you can include text here as a filler for the field in the edit form.